.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
   directories:
     - node_modules
 install:
-  - npm install
+  - npm ci
```

Comment:
I'm missing the "why" here in the commit description

Reply:
I don't think this should even have a package-lock, we're consumed mostly as a library, right?